meetings/notes/2023 - June 6th.md

```diff
@@ -0,0 +1,29 @@
+## Meeting Details
+
+- **Date/Time:** June 6th, 2023 @ 5:00pm UTC / 10:00am PDT
+- **Location:** [Discord SIG-Release Voice Room](https://discord.gg/Z2bzwCRJEz)
+- **Moderator**: @tonybalandiuk
+- **Note Taker**: @vincentvincent
+- **Agenda**: https://github.com/o3de/sig-release/issues/193
+
+## SIG Updates
+- Feature grid will still be part of the O3DE release deliverable as decided in the Joint TSC meeting. You can read the note [here](https://github.com/o3de/sig-release/issues/191#issuecomment-1568709344).
+
+## Meeting Notes
+1. Features grid
+   1. Will it be holding up a release? Since it's a documentation, this doesn't hold up a release
+   2. SIG release would like understand better on the features grid usage before we collectively decided to go with the features grid as release deliverable. The main motivation is so we can make sure the busy work is worth the SIGs time.
+   3. @tonybalandiuk to follow up with the SIG Docs Community about the features grid statisticts and report back into the Discord SIG release channel.
+2. On the Join TSC meeting on May 30th, 2023, Royal proposes to have the retrospective session together with TSC and other SIG members during one of the office hours. The agenda for today is to decide the following action steps, whether to hold the session with other members or do it ourselves.
+   1. SIG Release agrees with this.
+   2. @tonybalandiuk to pick one of the TSC meeting hours and then gets approval from TSC to run the retrospective session on the meeting.
+3. Decides when we are going to decide the next release date.
+   1. The next release will be harder as Amazon members won't participate much in the next release. The next big question is the QA resources.
+   2. @joe will talk with Nicole about this and report back the result.
+4. SIG Release Election
+   1. @vincent to figure out the election process and report back in the SIG release channel.
+
+## Action Items
```

Comment:
1 more action item: 
tonybalandiuk to pick one of the TSC meeting hours and then gets approval from TSC to run the retrospective session on the meeting.

Reply:
Thanks for catching the missing action items! I'll make updates today

Reply:
Done.